### DISCUSSION TITLE
Two way links

### DISCUSSION THREAD

**Discussion by Laaronhooper** [nvim-neorg/neorg#1028] [Q&A]
Hi,

Just wondering how or if there is a way have two way binding on notes. E.g. If I have a 'recipe' note and in that note have a link to a 'sandwich' note. I can move to the sandwich note by hitting enter on  {:sandwich:} but how do I jump back to the 'recipe' note?

Thanks in advance.

**Comment by max397574** [comment]
`<c-o>`

**Comment by benatouba** [comment]
Not an answer related directly to neorg but you could use <C-o> to move out/back in your jumplist. Other than that, your buffers will still be open, so you could cycle through buffers or use ":Telescope buffers" (if installed). 

I hope this helps even though not directly what youre looking for
EDIT: Someone was quicker.

**Reply by benatouba** [reply]
That said, it would be nice to have something similar to the (Obsidian backlinks plugin)[https://help.obsidian.md/Plugins/Backlinks] functionality.
